keyboards/coseyfannitutti/discipline/readme.md

```diff
@@ -0,0 +1,15 @@
+# DISCIPLINE
+
+![discipline](https://i.imgur.com/RbV5I0g.jpg)
+
+A 65% keyboard that can be assembled with only through hole components, including usb type-c
+
+Keyboard Maintainer: [coseyfannitutti](https://github.com/coseyfannitutti)  
+Hardware Supported: DISCIPLINE, atmega32a  
+Hardware Availability: [cftkb.com](http://www.cftkb.com), [GitHub](https://github.com/coseyfannitutti/discipline)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make coseyfannitutti/discipline:program
```

Comment:
This is actually `<keyboard>:<keymap>:<program>`, with `<program>` being optional.

```suggestion
    make coseyfannitutti/discipline:default:program
```